data/sqlutil/macros.go

```diff
@@ -132,6 +134,35 @@ func trimAll(s []string) []string {
 	return r
 }
 
+func parseArgs(argStr string) ([]string, error) {
+	parsed := []string{}
+	nested := 0
+	argStart := 0
+
+	for i := 0; i < len(argStr); i++ {
+		switch argStr[i] {
+		case '(':
+			nested++
+		case ')':
+			if nested == 0 {
```

Comment:
In actuality this one shouldn't happen, because `getMacroMatches` automatically cuts off the macro when it runs into an extra closing parenthesis.